File: ServerSpace/src/main/java/Models/General/UnidadMedidaModel.java
package Models.General;

import java.sql.Timestamp;

import com.fasterxml.jackson.annotation.JsonProperty;

import EntityLayer.UnidadMedidaEntity;

public class UnidadMedidaModel {
    

    public UnidadMedidaModel() {
        this.unidadMedidaId = 0;
        this.codigo = "";
        this.codigoComercial = "";
        this.nombre = "";
        this.fechaRegistro = null;
        this.codUsuario = "";
        this.estadoRegistro = false;
    }

    public UnidadMedidaModel(UnidadMedidaEntity Ent) {
        this.unidadMedidaId = Ent.getUnidadMedidaId();
        this.codigo = Ent.getCodigo();
        this.codigoComercial = Ent.getCodigoComercial();
        this.nombre = Ent.getNombre();
        this.fechaRegistro = Ent.getFechaRegistro();
        this.codUsuario = Ent.getCodUsuario();
        this.estadoRegistro = Ent.getEstadoRegistro();
    }


    @JsonProperty("UnidadMedidaId")
    private int unidadMedidaId = 0;
    public int getUnidadMedidaId() {
        return unidadMedidaId;
    }

    public void setUnidadMedidaId(int unidadMedidaId) {
        this.unidadMedidaId = unidadMedidaId;
    }

    
    @JsonProperty("Codigo")
    private String codigo = "";
    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    @JsonProperty("CodigoComercial")
    private String codigoComercial = "";
    public String getCodigoComercial() {
        return codigoComercial;
    }

    public void setCodigoComercial(String codigoComercial) {
        this.codigoComercial = codigoComercial;
    }

    @JsonProperty("Nombre")
    private String nombre = "";
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }


    @JsonProperty("FechaRegistro")
    private Timestamp fechaRegistro = null;
    public Timestamp getFechaRegistro() {
        return fechaRegistro;
    }

    public void setFechaRegistro(Timestamp fechaRegistro) {
        this.fechaRegistro = fechaRegistro;
    }

    @JsonProperty("CodUsuario")
    private String codUsuario = "";
    public String getCodUsuario() {
        return codUsuario;
    }

    public void setCodUsuario(String codUsuario) {
        this.codUsuario = codUsuario;
    }


    @JsonProperty("EstadoRegistro")
    private boolean estadoRegistro = false;
    public boolean getEstadoRegistro() {
        return estadoRegistro;
    }

    public void setEstadoRegistro(boolean estadoRegistro) {
        this.estadoRegistro = estadoRegistro;
    }
}

File: SharedApi/Space/Space.Server/Model/General/UnidadMedidaModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server.Model.General
{
    public class UnidadMedidaModel
    {
        public UnidadMedidaModel()
        {

            UnidadMedidaId = 0;
            Codigo = String.Empty;
            CodigoComercial = String.Empty;
            Nombre = String.Empty;
            FechaRegistro = DateTime.Now;
            CodUsuario = String.Empty;
            EstadoRegistro = true;

        }

        public UnidadMedidaModel(UnidadMedidadEntity Ent)
        {
            this.UnidadMedidaId = Ent.UnidadMedidaId;
            this.Codigo = Ent.Codigo;
            this.CodigoComercial = Ent.CodigoComercial;
            this.Nombre = Ent.Nombre;
            this.FechaRegistro = Ent.FechaRegistro;
            this.CodUsuario = Ent.CodUsuario;
            this.EstadoRegistro = Ent.EstadoRegistro;

        }
        [JsonPropertyName("UnidadMedidaId")]
        public Int32 UnidadMedidaId { get; set; }
        [JsonPropertyName("Codigo")]
        public String Codigo { get; set; }
        [JsonPropertyName("CodigoComercial")]
        public String CodigoComercial { get; set; }
        [JsonPropertyName("Nombre")]
        public String Nombre { get; set; }
        [JsonPropertyName("FechaRegistro")]
        public DateTime FechaRegistro { get; set; }
        [JsonPropertyName("CodUsuario")]
        public String CodUsuario { get; set; }
        [JsonPropertyName("EstadoRegistro")]
        public Boolean EstadoRegistro { get; set; }
    }
}
